RELEASE_NOTES.md

```diff
@@ -2,6 +2,281 @@
+- Reject bundles containing duplicated filenames (#1454)
+- Add manual GitLab action to release to staging (#1457)
+- Bump awscli to 1.15.0 or greater (#1462)
+- Add Index Design and search limitations documentation (#1461)
+- Switch to the constants defined in dcplib (#1465)
+- Add a bundle with new layout to test fixtures (#1468)
+- Use folded block YAML style for docs in swagger (#1470)
+- Check dcp-int status on GitLab (#1472)
+- Use different staging folders for building requiremnts. (#1464)
+- [Refactor] Changed TombstoneID to BundleTombstoneID (#1452)
+- Move errcodes out of the catch-all default error handler. (#1482)
+- Change illegal version to a 400. (#1483)
+- Support new bundle layout (#1469)
+- Fix shape_descriptor bug causing failed deploy
+- Upgrade to dcplib 1.3.2 (#1478)
+- Explicitly catch a malformed version error for PATCH /collections (#1485)
+- Refresh all requirements. (#1476)
+- Remove dss-chalice script
+- Deploy before performing integration tests (#1494)
+- Add force-release buttons to GitLab (#1489)
+- Update indexer test to new fixture (#1500)
+- Update fixture in tests/test_indexer.py (#1505)
+- Update smoketest bundle fixture (#1504)
+- Remove duplicated indexer test code (#1501)
+- Don't mark job as failed when not releasing to int (#1512)
+- Delete old bundle format fixtures (#1513)
+- Interpolate cli endpoint (#1502)
+- Improve error response documentation for PATCH /collections (#1490)
+- Checkout from GitHub with token for releases (#1510)
+- Remove unsused sample queries (#1503)
+- Incorporate terraform workflow into daemon deployment (#1425)
+- Update fixture for tests/test_api.py (#1514)
+- create blob TTL env vars
+- Remove unused test_api fixture. (#1519)
+- Add missing quotes to fix tf scripts
+- Add DSS_BLOB_TTL_DAYS to Terraform variables (#1526)
+- Release to integration requires manual action. (#1525)
+- Add link to CONTRIBUTING.md in README.md (#1528)
+- Add test for bundle delete notifications (#1527)
+- Prevent object modification rate limit touch test files (#1534)
+- Put the shape descriptor in the ES index doc (#1529)
+- Direct urls should point to checkout bucket (#1539)
+- Add `--no-deploy` flag to scripts/release.sh (#1537)
+- Refactor gitlab jobs (#1540)
+- files are checked out to checkout_bucket/blob (#1541)
+- directaccess and presignedurl should go to the checkout bucket (#1545)
+- Adding Debug log message when a collection is identified by the index
+- Remove unused dependency (moto) and its hazardous subdeps (#1544)
+- Consolodate bundle fixture (#1535)
+- Fix tests/test_smoketest.py typo (#1546)
+- Using Auth0 for authentication (#1542)
+- adding subtests to test_bundle (#1543)
+- get_file performs synchronously when possible (#1347)
+- get_bundle only performs checkout when necessary (#1347)
+
+
+### Secondary Analysis
+
+#### Falcon
+- Stand up Falcon v0.1.3 in production, No workflow will be started at this time.
+V0.1.0 (This release is the first iteration of this new service):
```

Comment:
Actually it might be more clear to remove all of the detailed release notes and only remain the first sentence `Standup .... this time`?

Reply:
fixed